### ISSUE TITLE
Slideshow Block: Allow arranging images by drag-n-drop

### ISSUE THREAD

**Issue by otpidusprime** [Automattic/jetpack#17112]
**This is a feature request.**

Received this feedback from a WPcom user requesting to be able to arrange images by drag-n-drop instead of clicking "Edit" and selecting images in a sequence.

I believe this is much more intuitive than the current way or arranging them, especially when there is a large number of images in the Slideshow.

One way to do this, that I had in mind, is when we click on "Edit" on the Slideshow Block, we can show the images that are added on the Block — allowing users to drag them in the order they want.

**Ticket:** 3300333-zen

**Comment by chaitanyamsv** [comment]
A customer asked for it in 26040205-hc

**Comment by kriskarkoski** [comment]
28934706-hc

**Comment by samiff** [comment]
@kriskarkoski When creating/adding a new Slideshow (Gallery), you do have to click to select the images initially, but once on the `Edit Gallery` screen you should be able to drag/drop the images:

![Markup on 2021-06-16 at 16:37:32](https://user-images.githubusercontent.com/15803018/122303835-6ac16000-cec1-11eb-88c6-3762f5d71803.png)

Were you noticing a different behavior?

**Comment by kriskarkoski** [comment]
Yep, I was ending up on the main media UI where you click the images you want to include and they're numbered and the only workaround I could find was clicking to remove an image and then clicking again to re-add it at the back. 

But yesterday I helped a user with the same block and got the UI you see, and I also see that now on this user's site so I'm confused how I got into the other UI to begin with but if I see it again I'll grab screenshots/URL paths.
